django_netjsonconfig/models.py

```diff
@@ -73,3 +80,18 @@ class VpnClient(AbstractVpnClient):
     """
     class Meta(AbstractVpnClient.Meta):
         abstract = False
+
+
+class TemplateSubscription(AbstractTemplateSubscription):
+    """
+    Concrete Template Subscription model
+    """
+    class Meta(AbstractTemplateSubscription.Meta):
+        abstract = False
+
+
+@periodic_task(run_every=crontab(minute='0', hour='0'))
```

Comment:
I don't know this approach well, with this approach do we need to run celery beat or not?
the periodicity needs to be configurable, I'd do it with celery beat.

Reply:
I think it's better to use the `CELERYBEAT_SCHEDULE` setting to define the periodicity